ui/src/components/system/Services.vue

```diff
@@ -75,12 +105,36 @@
         <td class="fancy">
           <span :class="['fa', props.row.running ? 'fa-check green' : 'fa-times red']"></span>
         </td>
+        <td class="fancy">
+          <span
+            v-for="(zone, i) in props.row.ports.access.split(',')"
+            v-bind:key="i"
+            :class="['label', 'label-info', defaultZones.includes(zone) ? 'bg-' + zone : 'bg-missing', 'pad-left-right-sm']"
+          >{{ zone }}</span>
+          <!-- show "localhost" for network services with access=<none> -->
+          <span
+            v-if="!props.row.ports.access && (props.row.ports.TCP.length > 0 ||
+            props.row.ports.UDP.length > 0)"
+            class="label label-info bg-gray pad-left-right-sm"
+          >localhost</span>
+        </td>
+        <td class="fancy">
+          <span v-if="props.row.ports.TCP.length">
+            <b>TCP:</b>
+            {{ props.row.ports.TCP.join(', ') }}&nbsp;&nbsp;
```

Comment:
if you want to create a space, we can add a `<br/>` tag after the `</span>` close tag